articles/azure-maps/drawing-package-guide.md

```diff
@@ -315,13 +315,6 @@ Defining text properties enables you to associate text entities that fall inside
 
 :::image type="content" source="./media/creator-indoor-maps/onboarding-tool/dwg-layers.png" alt-text="Screenshot showing the 'create a new manifest' screen of the onboarding tool.":::
 
-> [!IMPORTANT]
-> Wayfinding support for `Drawing Package 2.0` will be available soon. The following feature class should be defined (not case sensitive) in order to use [wayfinding]. `Wall` will be treated as an obstruction for a given path request. `Stair` and `Elevator` will be treated as level connectors to navigate across floors:
```

Comment:
We don't want to remove this whole piece, just Wayfinding support for `Drawing Package 2.0` will be available soon.